fuse_examples/segmentation/create_dataset.py

```diff
@@ -0,0 +1,169 @@
+# from fastai.vision import *
+import pydicom
+from pathlib import Path
+import pandas as pd
+from tqdm import tqdm as progress_bar
+import PIL
+import numpy as np
+import matplotlib.pylab as plt
+# from mask_functions import *
+# Here out RLE encoding is little bit different
+
+import sys
+sys.path.append('JustImage/')
+sys.path.append('JustImage/CV')
+from JustImage.CV.Binarization import bin_mosaic1, bin_mosaic3
+
+main_out_path = 'data' 
+dataset_path = 'siim'
+
+"""
+download dataset from - 
+https://www.kaggle.com/seesee/siim-train-test
+
+The path to the extracted data should be updated in the <dataset_path> variable.
+The output images will be stored at <main_out_path>.
+"""
+
+def create_mosaic(im, DEBUG=False):
+    # im = im.astype(np.uint8)
+    im = np.asarray(im)
+    org_min = np.min(im)
+    org_max = np.max(im)
+
+    _, im_after = bin_mosaic1(im, tiles=(8, 8), locality_size=8)
+    
+    s = (im_after - np.min(im_after)) / (np.max(im_after) - np.min(im_after))
+    out_im = s * (org_max - org_min) + org_min
+    out_im = out_im.astype(np.uint8)
+
+    _, im_after = bin_mosaic1(im, tiles=(16, 16), locality_size=8)
+    
+    s = (im_after - np.min(im_after)) / (np.max(im_after) - np.min(im_after))
+    out_im2 = s * (org_max - org_min) + org_min
+    out_im2 = out_im2.astype(np.uint8)
+
+    rgb = np.stack([im, out_im, out_im2], axis=-1)
+
+    if DEBUG:
```

Comment:
Let's remove debug code - it should easy to follow example.
Let's use visualizer if required,